modules/store/src/utils.ts

```diff
@@ -107,6 +115,7 @@ export function createFeatureReducerFactory<T, V extends Action = Action>(
       : (r: ActionReducer<T, V>) => r;
 
   return (reducer: ActionReducer<T, V>, initialState?: T) => {
+    // I think this is also bug, shouldn't it be: reducerFactory(reducer, initialState)?
```

Comment:
This is quite rare... 
It should affect only users that use `reducerFactory` for feature and pass reducer as function instead of dict.

Reply:
No it is not.
This is a reducer factory, it accepts one reducer as argument.
We wrap this reducer with our reducer and simply pass the arguments received, with the extra check on the state, to the "real" reducer.

Reply:
Ok I see, you'r right...
I thought this was a reducerFactory that possibly comes from the user,
I will remove the comment :)